docs/modeling-your-data/modeling-your-data-with-dbt/index.md

```diff
@@ -67,5 +68,12 @@ The supported data warehouses per version can be seen below:
 | ----------------------------- | -------------------- | ----------------- | -------- | ---------- | -------- | --------- | -------- |
 | 0.1.0                         | >=0.9.0 to <1.0.0    | >=1.0.0 to <2.0.0 | ✅        | ✅        | ❌        | ✅        | ❌        |
 
+</TabItem>
+<TabItem value="ecommerce" label="Snowplow E-commerce">
+
+| snowplow-normalize version | snowplow-web version | dbt versions      | BigQuery | Databricks | Redshift | Snowflake | Postgres |
```

Comment:
Should this say ecommerce and doesn't depend on the web version right? (I need to remove the web dependency from the normalize table...)